### PR TITLE
Fixing graph root padding and truncating long queries in execution plan

### PR DESCRIPTION
**Pull request by aasimkhan30**
After:
image

Before:
image

Issue: #18961

### CODE REVIEW

File: src/sql/workbench/contrib/executionPlan/browser/executionPlanViewHeader.ts [hunk 1]
```diff
@@ -88,6 +88,7 @@ export class ExecutionPlanViewHeader {
 	private renderQueryText(): void {
 		if (this._query) {
 			this._queryContainer.innerText = this._query;
+			this._queryContainer.title = this._query;
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/executionPlan/browser/executionPlanViewHeader.ts:91]: This makes it show the full query as a tooltip right?

Is there a way for the user to copy the query text easily? Might be a feature to consider adding later

File: src/sql/workbench/contrib/executionPlan/browser/media/executionPlan.css [hunk 2]
```diff
@@ -33,7 +33,7 @@
 The actual sash element constructed by code. Important is used here because the width of the sash is fixed.
 However we always want it to be the width of the container it is resizing.
 */
-.eps-container .execution-plan-sash > div {
+.eps-container .execution-plan-sash>div {
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/executionPlan/browser/media/executionPlan.css:36]: nit : But generally it's best to leave the spaces. Makes it easier to read and parse the selector

**Review by Charles-Gagnon — APPROVED** [review]

File: src/sql/workbench/contrib/executionPlan/browser/executionPlanViewHeader.ts (hunk 1 above)
Comment (aasimkhan30) [reply src/sql/workbench/contrib/executionPlan/browser/executionPlanViewHeader.ts:91]: They have an option to open query in a new query window for now. 😃

**Review by lewis-sanchez — APPROVED** [review]

File: src/sql/workbench/contrib/executionPlan/browser/media/executionPlan.css (hunk 2 above)
Comment (aasimkhan30) [reply src/sql/workbench/contrib/executionPlan/browser/media/executionPlan.css:36]: It seems that auto format got rid of it

Comment (Charles-Gagnon) [reply src/sql/workbench/contrib/executionPlan/browser/media/executionPlan.css:36]: Huh, annoying. Ok well if the formatter is doing it then fine to leave it like that then.

**Review by erinstellato-ms — COMMENTED** [review]
Looks good visually - real estate is critical on the screen, so giving folks some lines back for larger plans is helpful.  I appreciate you making the change so quickly.

**Review by kburtram — APPROVED** [review]

**Pull request merged**